specification/eventgrid/resource-manager/readme.md

```diff
@@ -29,6 +29,15 @@ openapi-type: arm
 tag: package-2020-06
```

Comment:
Will this version be generated for SDK? if yes. default tag should change to package-2020-10-preview, so we can have correct sdk validation during PR review phase

Reply:
Yes we will have SDK for this new api version. I have made the changes.